src/main/kotlin/com/github/bartimaeusnek/modmixins/core/LoadingConfig.kt

```diff
@@ -15,7 +15,7 @@ object LoadingConfig {
         fixZtonesNetworkVulnerability = config["fixes", "fixZtonesNetworkVulnerability", true, "Fixes Ztones Network Vulnerability"].boolean
 
         RailcraftJarName = config["jars", "Railcraft Jar Name","Railcraft_1.7.10-9.12.2.1.jar", "Name of the Railcraft Jar"].string
-        GalacticraftJarName = config["jars", "Galacticraft Jar Name","Galacticraft-1.7-3.0.14-GTNH-full.jar", "Name of the Galacticraft Jar"].string
+        GalacticraftJarName = config["jars", "Galacticraft Jar Name","Galacticraft-1.7-3.0.18-GTNH-full.jar", "Name of the Galacticraft Jar"].string
```

Comment:
@bartimaeusnek this seems a bit brittle, any ideas on a better way of doing it? (and the stacktrace was useless)

Reply:
This is actually a config value and we don't need to recompile this mod every release. It's still somewhat weird to be honest though

Reply:
@mitchej123 well potentially it could be recoded so it applies a regex to find any Galacticraft.*\.jar files. But sadly, it needs to manually load the files in advance, and unload them aswell, since Mixins isn't stripping hard deps during compilation, other than ASM loaders.

Reply:
Can you have it error out if it can't find the file?  A less cryptic error would be helpful.

The regex sounds a better idea than hardcoding the file names.  Curious what you mean by "since Mixins isn't stripping hard deps during compilation, other than ASM loaders."